content/docs/getting-started/db.md

```diff
@@ -188,7 +201,7 @@ just panic if you request a different path. Next up let's generate the id of
 the row that we're going to look up:
 
 ```rust,ignore
-let random_id = rand::thread_rng().gen_range(0, 10_000);
+let random_id = rand::thread_rng().gen_range(1, 5);
```

Comment:
In Postgres the `serial` type starts at 1 and the example has 5 rows in the `greetings` table.

I briefly considered adding a query when the program starts, to find the number of rows and use it as an upper bound of the random number, but I concluded that it might make the example more complex than necessary.